pkg/deploy/internal/setting/setting.go

```diff
@@ -52,7 +52,7 @@ func Deploy(ctx context.Context, settingsClient client.SettingsClient, propertie
 		Content:        []byte(renderedConfig),
 		OriginObjectId: c.OriginObjectId,
 	}
-	upsertOptions := makeUpsertOptions(c)
+	upsertOptions := makeUpsertOptions(c, insertAfter)
```

Comment:
stupid functional question: how does the API behave if the config ID to insert after does not exist? Just ignore it I hope?

Reply:
and how does it behave if it is not an object ID? With the specific parameter name and no extra validation that I can see so far, this might be any type of param.

Reply:
> Just ignore it I hope?
yes